src/common/modals/ModpackDescription.js

```diff
@@ -11,6 +14,35 @@ import CloseButton from '../components/CloseButton';
 import { closeModal } from '../reducers/modals/actions';
 import { FORGE, CURSEFORGE_URL } from '../utils/constants';
 
+const formatNumber = number => {
+  // Alter numbers larger than 1k
+  if (number >= 1e3) {
```

Comment:
Codacy found an issue: [The numeric literal '1e3' will have at different value at runtime.](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5870474)